.travis.yml

```diff
@@ -19,8 +15,6 @@ matrix:
     - python: "3.7"
       env: TOXENV=py37
 
-    # Linters
-
     - python: "3.7"
```

Comment:
Just double checking that python 3.7 is right.

Reply:
Good catch! It probably doesn't matter much in the grand scheme of things because it's only linting, but good to be consistent!